src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEModuleSymbol.cs

```diff
@@ -702,5 +708,43 @@ internal IEnumerable<NamedTypeSymbol> GetForwardedTypes()
         }
 
         public override ModuleMetadata GetMetadata() => _module.GetNonDisposableMetadata();
+
+        internal bool ShouldDecodeNullableAttributes(Symbol symbol)
+        {
+            Debug.Assert(!(symbol is null));
+            Debug.Assert(symbol.IsDefinition);
+            Debug.Assert((object)symbol.ContainingModule == this);
+
+            if (_lazyNullableMemberMetadata == NullableMemberMetadata.Unknown)
+            {
+                _lazyNullableMemberMetadata = GetAttributes().IndexOfAttribute(this, AttributeDescription.NullablePublicOnlyAttribute) >= 0 ?
+                    (_assemblySymbol.GetAttributes().IndexOfAttribute(this, AttributeDescription.InternalsVisibleToAttribute) >= 0 ?
```

Comment:
>_assemblySymbol [](start = 21, length = 15)

This feels wrong, ```_assemblySymbol``` could be a source assembly symbol that has nothing to do with what attributes we have in the module.